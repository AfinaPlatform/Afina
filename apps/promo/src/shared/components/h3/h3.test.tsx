import H3 from "./h3";
import { render } from "@testing-library/react";

describe("shared/h3", () => {
  it("render h3 html tag", () => {
    const { container } = render(<H3>Hello</H3>);
    expect(container.querySelector("h3")?.tagName).toBe("H3");
  });

  it("snapshot", () => {
    const { asFragment } = render(<H3>Hello</H3>);
    expect(asFragment()).toMatchSnapshot();
  });
});
