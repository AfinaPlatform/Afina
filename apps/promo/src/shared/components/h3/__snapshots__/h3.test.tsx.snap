// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shared/h3 snapshot 1`] = `
<DocumentFragment>
  <h3
    class="root"
  >
    Hello
  </h3>
</DocumentFragment>
`;
